components/admin/lessons/AdminLessonInputs.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react'
+import { useAddModuleMutation } from '../../../graphql'
+import { formChange } from '../../../helpers/formChange'
+import { FormCard, Option } from '../../FormCard'
+import { AlertFillIcon, CheckCircleIcon } from '@primer/octicons-react'
+import styles from '../../../scss/adminLessonInputs.module.scss'
+import { Spinner } from 'react-bootstrap'
+import { GraphQLError } from 'graphql-request/dist/types'
+
+type Props = {
+  lesson: { title: string; id: number }
+}
+
+enum Error {
+  INVALID_DATA = 'missing module name or description'
```

Comment:
The convention for typescript enums is to use pascalCase
https://www.typescriptlang.org/docs/handbook/enums.html